toolchain/check/convert.cpp

```diff
@@ -183,15 +183,16 @@ static auto ConvertAggregateElement(
     SemIR::TypeId src_elem_type,
     llvm::ArrayRef<SemIR::InstId> src_literal_elems,
     ConversionTarget::Kind kind, SemIR::InstId target_id,
-    SemIR::TypeId target_elem_type, PendingBlock* target_block, size_t i) {
+    SemIR::TypeId target_elem_type, PendingBlock* target_block,
+    size_t src_field_index, size_t dest_field_index) {
```

Comment:
Looks like the comment needs to be updated to reflect these changes.

Reply:
Ah, yep - done in https://github.com/carbon-language/carbon-lang/pull/4565/commits/fafd925c8a353c1b0a829845c86f3920d8071b1e